errors.go

```diff
@@ -7,6 +7,7 @@ import (
 	"net/http"
 	"strings"
 
+	"github.com/golang/protobuf/proto"
```

Comment:
this import path was recently deprecated: https://github.com/golang/protobuf/pull/1306/files